pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,132 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+    """Manages the parameters used for computing the mean/variance."""
+    eps: float
+    delta: float
+    low: float
+    high: float
+    max_partitions_contributed: int
+    max_contributions_per_partition: int
+    noise_kind: pipeline_dp.NoiseKind  # Laplace or Gaussian
+
+    def l0_sensitivity(self):
+        return self.max_partitions_contributed
+
+    def linf_sensitivity(self, metric):
+        return self.max_contributions_per_partition * {
```

Comment:
This is a pretty cool trick, but I'd suggest to have something line

if metric == COUNT:
   return ...
if metric == SUM:
   return
raise ValueError(...)

Clarification: it's slightly longer, but it's more readable and it's easy to see what's l_inf of each of metrics